### PR TITLE
added dark mode

### PR DESCRIPTION
**Pull request by ashuydv**
I added dark mode as asked, in all the  pages and also in the resposive feature. 
Do review it,

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (anushbhatia) [inline .vscode/settings.json:2]: Remove this file. Rest looks good to me. 
@smaranjitghose any, reviews?

**Review by anushbhatia — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (smaranjitghose) [reply .vscode/settings.json:2]: Yep. It's cool!

**Comment by smaranjitghose** [comment]
@anushbhatia @ashuydv  There is one issue though. If you are at a page X, you change the theme to dark, then visit page Y, the theme switches back to X. However, it should remain the same.

Moreover, I would suggest to make the default theme as dark

**Comment by anushbhatia** [comment]
> @ashuydv
> 
> ## Feedback:
> * There is a small bug If you are on page X, you change the theme to dark, then visit page Y, the theme switches back to X. However, it should remain the same.
> * Moreover, I would suggest making the default theme as the system default.
> * The research page cards especially the buttons look awkward and need to be rectified
> * Experience Page lowermost cards (Mentorship) need to be fixed
> * The color of the scroll bar should be light and dark shades of purple for light and dark themes
> * Home Page: Medium and Kaggle Icons are faded
> 
> cc: @anushbhatia

I think this can be done in the next PR, cause this is enough for the first one. Rest he can enhancement one by one. what say @ashuydv @smaranjitghose

**Comment by ashuydv** [comment]
Yes, I'll be trying to make it common , I'll try to make a PR as soon as possible. I noted all the points

File: research.html [hunk 2]
```diff
@@ -1,55 +1,54 @@
...
 <head>
-  <title>John Doe | Research</title>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="keywords"
-    content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-  <meta name="description"
-    content="This project serves free and beautiful portfolio template for everyone. The sole aim of this project is to help all those who are not sure or face difficulties while trying to create the perfect portfolio for themselves. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <title>John Doe | Research</title>
+    <meta charset="UTF-8">
```
Comment (smaranjitghose) [inline research.html:6]: Avoid indentation and linting related changes for now. Else others would face merge conflicts

**Comment by smaranjitghose** [comment]
@ashuydv Raise a new Issue for the above points and split the work into 2-3 PRs.

**Comment by ashuydv** [comment]
okay,

**Pull request merged**